integration/templates/ingress_controller_values.go

```diff
@@ -0,0 +1,9 @@
+// +build k8srequired
+
+package templates
+
+// IngressControllerValues sets legacy to true so the controller service uses
+// node ports.
```

Comment:
The chart defaults to creating a load balancer for CAPI clusters. For KIND we fall back to legacy and use a node port. This is so the helm release test passes.